common/src/main/java/com/skytouch/common/model/Product.java

```diff
@@ -0,0 +1,90 @@
+package com.skytouch.common.model;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+
+public class Product implements Serializable {
```

Comment:
can `Product` class be final?

Reply:
yes  it can, when a class is not designed for inheritance, this should be final.